Std/Data/List/Lemmas.lean

```diff
@@ -524,10 +524,25 @@ theorem get?_eq_get : ∀ {l : List α} {n} h, l.get? n = some (get l ⟨n, h⟩
   | _ :: _, 0, _ => rfl
   | _ :: l, _+1, _ => get?_eq_get (l := l) _
 
+theorem get!_eq_get : ∀ {l : List α} {n} h, l.get? n = some (get l ⟨n, h⟩)
```

Comment:
In this case, the explicit parameter `h` should have a type to guide usage. The type of `h` can be inferred by Lean but a human reading the code could use the hint that `h : n < l.length`.

Reply:
done, also in the lemma above for consistency